vm/src/obj/objint.rs

```diff
@@ -31,6 +31,26 @@ impl IntoPyObject for usize {
     }
 }
 
+impl TryFromObject for usize {
+    fn try_from_object(vm: &mut VirtualMachine, obj: PyObjectRef) -> PyResult<Self> {
+        // FIXME: don't use get_value
```

Comment:
(Why?)

Reply:
Right now it will panic if it receives a non-int.

```
>>>>> 'abcd'.startswith('b', '')
thread 'main' panicked at 'Inner error getting int [PyObj some rust value]', vm/src/obj/objint.rs:120:9
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

I could just add a `instanceof` check, but ultimately I want to delegate that to the `PyInt` payload's impl (which doesn't exist yet), like so:

```rust
impl TryFromObject for usize {
    fn try_from_object(vm: &mut VirtualMachine, obj: PyObjectRef) -> PyResult<Self> {
        match PyInt::try_from_object(vm, obj)?.to_usize() {
            Some(value) => Ok(value),
            None => Err(vm.new_overflow_error("Int value cannot fit into Rust usize".to_string())),
        }
    }
}
```